### PR TITLE
Monitor available entropy

### PR DESCRIPTION
**Pull request by gerardocorea92**
### Required for all PRs:

- [X] Signed [CLA](https://influxdata.com/community/cla/).
- [X] Associated README.md updated.
- [X] Has appropriate unit tests.

### CODE REVIEW

**Comment by danielnelson** [comment]
closes #2647

File: plugins/inputs/system/kernel.go [hunk 1]
```diff
@@ -33,13 +35,27 @@ func (k *Kernel) Description() string {
...
+
+	entropyString := string(entropyData)
+	entropyValue, err := strconv.ParseInt(strings.TrimSpace(entropyString), 10, 64)
+	if err != nil {
+		return err
+	}
+
 	fields := make(map[string]interface{})
 
+	fields["entropy_available"] = int64(entropyValue)
```
Comment (danielnelson) [inline plugins/inputs/system/kernel.go]: Let's just call it `entropy_avail` to match what is in proc.

**Comment by kyzh** [comment]
Is there anything that I can do to help this PR to be merged and shipped ?
I could bake a cake, but I'm a long way away from SF to deliver it to your office.

**Comment by hanej** [comment]
I have something like this to collect entropy in the mean time.

**telegraf.conf**
```
[[inputs.exec]]
commands = ["/etc/telegraf/collectors/collect_*.sh"]
data_format = "influx"
timeout = "10s"
```

**shell script**
```
#!/bin/bash

LOCATION=/etc/ansible/facts.d/location.fact

if [ ! -f $LOCATION ]; then
    exit 1
fi

DC=$($LOCATION | grep datacenter | awk '{print $2}' | sed -r 's/"(.*)".*/\1/')
ENV=$($LOCATION | grep '"environment"' | awk '{print $2}' | sed -r 's/"(.*)".*/\1/')
HOST=$(hostname -s)

if [ -z $DC ] || [ -z $ENV ]; then
    exit 1
fi

if [ ! -f /proc/sys/kernel/random/entropy_avail ]; then
    exit 1
fi

VALUE=$(cat /proc/sys/kernel/random/entropy_avail)

echo "system,dc=${DC},env=${ENV},host=${HOST} entropy=${VALUE}"
```

**Comment by danielnelson** [comment]
@gerardocorea92 Do you think you could fix the unit test?

**Comment by gerardocorea92** [comment]
@danielnelson pushed commit to fix CircleCI testing.

**Pull request merged**